packages/compass/src/main/window-manager.ts

```diff
@@ -193,7 +168,7 @@ function showConnectWindow(
    * Open all external links in the system's web browser.
    */
   window.webContents.setWindowOpenHandler((details) => {
-    void shell.openExternal(urlWithUtmParams(details.url));
```

Comment:
If this PR is not applicable to the places where Link component is not used, sholud we keep this for now? Are we planning a follow-up to address those places too?

Reply:
I'm planning on doing the TODOs in this PR.